package.json

```diff
@@ -58,6 +58,8 @@
     "@babel/polyfill": "^7.4.0",
     "axios": "^0.18.0",
     "body-parser": "^1.18.3",
+    "csv-write-stream": "^2.0.0",
```

Comment:
I see these new libraries added, but didn't see them reference in your files--are you using them?

Reply:
Good eye. I was originally using these in my seed script, but decided not to in the end. I've deleted them from my package now!